Header header
HSE_Object[] objects
sensor_msgs/Image image